### ISSUE TITLE
2 mp4 file when download one completed，there is no obvious indication that one of them has been downloaded.

### ISSUE THREAD

**Issue by JSZMonkey** [asapach/peerflix-server#164]
When a seed file contains 2 mp4 files, I only download  one. When one of the downloads is completed, there is no obvious indication that one of them has been downloaded.

**Comment by asapach** [comment]
Yeah, the only indication is that the checkbox next to the file becomes unchecked. It's the consequence of using [torrent-stream](https://github.com/mafintosh/torrent-stream) under the hood, because it doesn't provide any kind of api for this. I was thinking that instead of unchecking the checkbox I could try to make it checked and disabled, like this: 
![image](https://user-images.githubusercontent.com/1755336/50959926-33bb1900-14d5-11e9-9bcb-f3ecd0f2bf35.png)
Thoughts?

**Comment by JSZMonkey** [comment]
I think , don't add links without downloading, only have links to files when the file is downloaded.

**Comment by asapach** [comment]
Sorry, but I disagree:
> don't add links without downloading

The whole point of *streaming* torrents is that you don't need to *download* the whole file in order to access it. You can randomly seek inside the file that is partially downloaded.
> add a single file delete button

Unfortunately files cannot be deleted once they are downloaded. You can only delete the entire torrent. That's how torrent clients are supposed to work, because they need to be able to seed the downloaded chunks.
> add the automatic transcoding function

I've tried that a couple of years ago and it didn't work very reliably for real-time streaming, so I just gave up. I'm not planning to add any special handling for video files. I recommend using VLC to play the video files as network streams.

**Comment by nk932714** [comment]
@asapach can't we replace the tick mark with This type of downloading GIF ?
![wip-bouncing-arrow-animation](https://user-images.githubusercontent.com/29853208/72505495-0df1ee00-3866-11ea-8bc3-12b5e2aafaa6.gif)

and on downloading completion gif changes to this type of logo?
![image](https://user-images.githubusercontent.com/29853208/72505359-ca977f80-3865-11ea-9d2f-deda71ea7f7d.png)

**Comment by asapach** [comment]
@nk932714, the checkbox is meant to convey that the file is currently being downloaded (see [here](https://github.com/asapach/peerflix-server/blob/84447e850614a8e4694eed246a5e0f1ecbd26c9b/server/index.js#L49)): if it's checked then somebody is interested in it (you can either check it manually, or it will activate automatically when streaming), otherwise it's idle. Replacing it with a gif won't make any functional difference and will only annoy people who hate animated gifs.
I'll think about what can be done better when the download completes.
